packages/extension-base/src/background/handlers/Tabs.ts

```diff
@@ -145,7 +145,13 @@ export default class Tabs {
     const encodedWebsite = encodeURIComponent(phishingWebsite);
     const url = `${chrome.extension.getURL('index.html')}#${PHISHING_PAGE_REDIRECT}/${encodedWebsite}`;
 
-    chrome.tabs.update({ url });
+    chrome.tabs.query({ active: true, currentWindow: true, windowType: 'normal' }, (tabs) => {
```

Comment:
Shouldn' this be something like -

```js
chrome.tabs.query({ url: phishingWebsite }, (tabs) => {
  tabs.forEach((tab) => {
    chrome.tabs.update(tab.is, { url });
  });
});
```

So basically it checks all the available tabs for the url?

Reply:
The `tabs` here is an empty array, so it ends up opening the new phishing tab in background, without showing the phishing landing page.

I've tried many things in the query, but I am never able to get this specific website.

Also it's actually hard to debug because the `tab` ojects from the `tabs` array do not have any `url`...

```
      chrome.tabs.query({ currentWindow: true }, (tabs) => {
        console.log('tabs', tabs);
        tabs.forEach((tab) => {
          console.log('url', tab.url);
          chrome.tabs.update({ url });
        });
      });
```
returns with 4 tabs + opening the phishing website in a new tab:
> 5 url undefined

^ this shows that we should be able to query the tab with phishing website, but this means we need its id... which I don't know how to get as of now.

Reply:
TL;DR PITA